loops_lab.js

```diff
@@ -0,0 +1,256 @@
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+let num = 80
+while (num > 0) {
+console.log(num)
+  num -= 1
+}
+
+for (let num = 80; num > 0; num -= 1) {
+  console.log(num)
+}
+
+
+// 2. Write a while loop and a for loop that takes the variable "max", and iterates over all numbers from 0 to "max". For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+let max = 0
+while ( max <= 15) {
+if (max % 2 === 0){
+  console.log ( max + " is even ")
+} else {
+  console.log( max + " is odd ")
+}
+max += 1
+}
+
+for (let max = 0; max <= 15; max +=1) {
+  if (max % 2 ===0) {
+    console.log ( max + " is even ")
+  } else {
+    console.log (max + " is odd ")
+    }
+  }
+
+
+
+// ( originial questions # 2) Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+let num = 0
+while (num <= 70 && num >= 0) {
+if (num % 2 === 0){
+  console.log( num + ' even number ')
+} else {
+  console.log( num + ' odd number ')
+}
+num += 1
+}
+//
+for (let num = 0; num <= 70; num += 1) {
+if (num % 2 === 0) {
+  console.log (num + ' even number ')
+} else {
+  console.log( num + ' odd number ')
+  }
+}
+//
+// 3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//         Hint
+//       Find the final number and increment the loop by 9.
+//     </details>
+
+let x = 0
+while (x <= 12) {
+console.log( x + " * 9 =  " + x * 9)
+// console.log (x  * 9)
+num +=1
+}
+
+for ( let x = 0; x <= 12; x += 1) {
+  console.log (x + " * 9 = " + x * 9)
+}
+//
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, exclusive.
+
+let a = 1
+while ( a <= 100) {
+if (a % 5 === 0 && a % 10 === 5) {
+  console.log(a)
+  }
+a ++
+}
+
+// 5. Without running/executing your code, how many times will the loop below run? Explain why.
+// let i = 5;
+//
+// while (i > 3) {
+//     i += 1
+// }
+
+// Your explanation here
+This code is executing by one and without stopping any number greater than three but since its missing the console.log it cannot run.
+It cannot demostrate that if the boolean expression is true.
+
+//  6. (0riginal #4 ) Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+//
+
+let num = 0
```

Comment:
This code would give me fizz and buzz numbers but not numbers that do not belong in both criteria (for example the answer should be: 
1
2
fizz
4
buzz
6
.....